packages/astro/src/cli/docs/open.ts

```diff
@@ -9,7 +9,6 @@ const getPlatformSpecificCommand = (): [string] | [string, string[]] => {
 	const isGitPod = Boolean(process.env.GITPOD_REPO_ROOT);
 	const platform = isGitPod ? 'gitpod' : process.platform;
 
-	// eslint-disable-next-line @typescript-eslint/switch-exhaustiveness-check
```

Comment:
This is causing the tests to fail

Reply:
But if we remove it, eslint complains that the switch check isn't exhaustive